### PR TITLE
add testcases for kubelet getters

### PR DESCRIPTION
**Pull request by ynqa**
**What type of PR is this?**

/kind cleanup

**What this PR does / why we need it**:

Add test cases for kubelet getters on `TestKubeletDirs`.

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by SergeyKanzhelev** [comment]
Welcome to k8s! Thank you for improving code coverage. Would you include test coverage for other getters. For instance, `getVolumeDevicePluginsDir`?

**Comment by ynqa** [comment]
> Would you include test coverage for other getters. For instance, `getVolumeDevicePluginsDir`?

@SergeyKanzhelev Added in https://github.com/kubernetes/kubernetes/pull/93723/commits/4357551ae3931a9ae488e691f3f6b0119ebbcae6. The rest of the cases will be tackled later.

**Comment by ynqa** [comment]
> Would you include test coverage for other getters. For instance, `getVolumeDevicePluginsDir`?

@SergeyKanzhelev Added the test cases for simple getters in https://github.com/kubernetes/kubernetes/pull/93723/commits/4357551ae3931a9ae488e691f3f6b0119ebbcae6. The rest will be tackled later.

File: pkg/kubelet/kubelet_getters_test.go [hunk 1]
```diff
@@ -63,7 +77,29 @@ func TestKubeletDirs(t *testing.T) {
...
+
 	got = kubelet.getPodContainerDir("abc123", "def456")
 	exp = filepath.Join(root, "pods/abc123/containers/def456")
 	assert.Equal(t, exp, got)
+
+	got = kubelet.getPodResourcesDir()
+	exp = filepath.Join(root, "pod-resources")
+	assert.Equal(t, exp, got)
+
+	// GetNodeConfig for containerManagerStub returns an empty node config
```
Comment (SergeyKanzhelev) [inline pkg/kubelet/kubelet_getters_test.go:96]: I don't think this test and the next one actually test anything. I would remove it or implement a proper stub that can return something meaningful to compare

Comment (ynqa) [reply pkg/kubelet/kubelet_getters_test.go:96]: > implement a proper stub that can return something meaningful to compare.

Thanks, sure. I will try it.

Comment (SergeyKanzhelev) [reply pkg/kubelet/kubelet_getters_test.go:96]: do you still want to implement it or simply remove tests that validate stubs?

Comment (ynqa) [reply pkg/kubelet/kubelet_getters_test.go]: I will try them on another PR. So deleted the tests in https://github.com/kubernetes/kubernetes/pull/93723/commits/7d58ead809e97b23bb8f9fba3cefb5ab8cfa81e4.

**Pull request merged**